### DISCUSSION TITLE
Seems to be confusion over tonic, root, bass

### DISCUSSION THREAD

**Discussion by staffanberglund** [tonaljs/tonal#376] [General]
Hi, I read through the docs and some issues and noticed these terms seem to have been confused in a few places. In the chord package specifically. 

- Tonic is the first scale degree, so the note C in the C major scale, the tonic chord is C major (or Cmaj7 etc) 
- Root is the note which a chord is built from, in a Cmaj7 this is C, in a Dm/A this is D
- And the bass note is the lowest note in a chord, so A in Dm/A.

So, if we are in the key of F minor, C7/E is the dominant chord, of which C is the root, E is the bass and Fm is the tonic chord.

Sorry if this was the wrong place to put this, I am new to GitHub!

**Comment by danigb** [comment]
Hi, thanks for the information. Yes, probably `root` should be called `bass`, but there's no easy way to change that without breaking change. Anyway, do you have a specific reliable source with that information? Also, do you have any suggestion on how to improve documentation?

**Reply by EricDiMarzio** [reply]
musictheory.net is a good source for information like this, but it's set up as an education website and not necessarily a wiki. If you're still looking for help with your documentation, I'm a musician/educator who recently switched to software development and I'm looking to get more involved with open source projects.
